pull-request.sh

```diff
@@ -136,7 +159,7 @@ main () {
             fi
             echo "Pull request title is ${PULL_REQUEST_TITLE}"
 
-            create_pull_request "${BRANCH}" "${PULL_REQUEST_BRANCH}" "${PULL_REQUEST_BODY}" "${PULL_REQUEST_TITLE}" "${PULL_REQUEST_DRAFT}"
+            create_pull_request "${BRANCH}" "${PULL_REQUEST_BRANCH}" "${PULL_REQUEST_BODY}" "${PULL_REQUEST_TITLE}" "${PULL_REQUEST_DRAFT}" "${PULL_REQUEST_MAINTAINER_CAN_MODIFY}" "${PULL_REQUEST_ASSIGNEE}"
 
```

Comment:
Where is `PULL_REQUEST_ASSIGNEE` used?

Reply:
in the issue update curl call - to assign this PR to someone for example the git.pusher.